Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN npm config set cafile "/app/DOIRootCA2.cer" \
 
 RUN node_modules/bower/bin/bower install --allow-root
 
+RUN nosetests --all-modules --exe
```

Comment:
I took it out and did a Docker build and the tests did run. Maybe try again without it?